waspc/src/Wasp/AppSpec/Core/Decl/JSON.hs

```diff
@@ -0,0 +1,28 @@
+{-# LANGUAGE TypeApplications #-}  -- Needed by generateFromJsonInstanceForDecl (TH)
+{-# OPTIONS_GHC -Wno-orphans #-}
```

Comment:
Because we put the instance defintiions ina  file wihch has neither class definition nor those type definitions. So it complains, because those are orphaned instances and that can be a problem in libraries, but not in application code really so we don't care about it, we have our reasons why we centralized it here.